precompile-patches/sp1.toml

```diff
@@ -1,6 +1,6 @@
 [patch.crates-io]
 sha2 = { git = "https://github.com/sp1-patches/RustCrypto-hashes", tag = "patch-sha2-0.10.8-sp1-4.0.0", package = "sha2" }
```

Comment:
@kevaundray, there're other precompiles available that aren't included here (e.g. `ecdsa`). 

Are the missing ones not here intentionally? Just checking that maybe we aren't missing any other one that can improve performance.